cloud_plugin_test.go

```diff
@@ -63,7 +63,7 @@ func TestInstallCloudPlugin(t *testing.T) {
 	}
 
 	for _, code := range []int{401, 403, 404, 412} {
-		server.code = code
+		_, client = gapiTestTools(t, code, "error")
```

Comment:
Given these are tests it's ok to ignore the server, but ideally we should still assign it to a variable and call `server.Close()`, otherwise we will be leaking resources.

Reply:
I've removed all of these from individual tests and now I'm doing a global `t.Cleanup`. The "server" is actually not needed in any test, so the mock function doesn't even need to return it